torch/ao/quantization/qconfig_mapping.py

```diff
@@ -120,7 +120,7 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
 
     return qconfig_mapping
 
-def get_default_qconfig_mapping(backend="fbgemm", version=0) -> QConfigMapping:
+def get_default_qconfig_mapping(backend="x86", version=0) -> QConfigMapping:
```

Comment:
Please update the code comment to mention default x86 backend.

Reply:
OK. It's done